packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -586,4 +588,35 @@ async function handler(input: CancelBookingInput) {
   } satisfies HandleCancelBookingResponse;
 }
 
+export type CancelRegularBookingData = CancelBookingInput["bookingData"];
+
+export type CancelBookingMeta = {
+  userId?: number;
+  platformClientId?: string;
+  platformRescheduleUrl?: string;
+  platformCancelUrl?: string;
+  platformBookingUrl?: string;
+  arePlatformEmailsEnabled?: boolean;
+};
```

Comment:
can we improve the type safety? currently all these fields are optional

Reply:
Yeah, I can take it up in a followup. Current goal is to move to DI without making unnecessary changes